### PR TITLE
Adjusted travis.yml and pom.xml files.

### PR DESCRIPTION
**Pull request by gbuh**
Adjusted travis.yml and pom.xml files.

### CODE REVIEW

**Comment by mf-code** [comment]
[5939] - y7ZpiDWyFPao

**Comment by mf-code** [comment]
[5939] - yUeTvgKXGQGz

**Comment by mf-code** [comment]
[5939] - bLNwTLfSuBvQ

**Comment by mf-code** [comment]
[5939] - tMDbK6I24XTI

**Comment by mf-code** [comment]
[5939] - HKh5QD0NKQJN

**Comment by mf-code** [comment]
[5939] - HaoolKayieRu

**Comment by mf-code** [comment]
[5939] - 27V2hOVozf7s

**Comment by mf-code** [comment]
[5939] - O036WoQWQfZ1

**Comment by mf-code** [comment]
[5939] - BG4nv12J0XKL

File: .travis.yml [hunk 1]
```diff
@@ -55,14 +57,22 @@ testsbootstrap: &testsbootstrap
     - mvn -ntp install -DskipTests
     - echo arguments=$CHROME_ARGS > "./$BOOTSTRAP_TESTS/src/test/resources/$CHROME_PROPERTIES"
     - mvn verify $WITH_PARAMS -pl $GROUP_ID:$BOOTSTRAP_TESTS
-testsangular: &testsangular
+testsangularunitcommon: &testsangularunitcommon
   stage: test
   name: Run tests - angular
```
Comment (BelAn97) [inline .travis.yml:62]: add profile name

File: .travis.yml [hunk 2]
```diff
@@ -55,14 +57,22 @@ testsbootstrap: &testsbootstrap
...
   <<: *cron_condition
   script:
     - mvn -ntp install -DskipTests
     - echo arguments=$CHROME_ARGS > "./$ANGULAR_TESTS/src/test/resources/$CHROME_PROPERTIES"
-    - mvn verify $WITH_PARAMS -pl $GROUP_ID:$ANGULAR_TESTS
+    - mvn verify $WITH_PARAMS $UNIT_COMMON -pl $GROUP_ID:$ANGULAR_TESTS
+testsangularcomplex: &testsangularcomplex
+  stage: test
+  name: Run tests - angular
```
Comment (BelAn97) [inline .travis.yml:70]: add profile name

**Comment by mf-code** [comment]
[5939] - B8U2Dmj6JxSY

**Comment by mf-code** [comment]
[5939] - GuTP4M7BefMy

**Comment by mf-code** [comment]
[5939] - Uqiy8bAVS8Uk

**Review by AlexSoftt — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (gbuh) [reply .travis.yml:62]: Done. Thank you.

File: .travis.yml (hunk 2 above)
Comment (gbuh) [reply .travis.yml:70]: Done. Thank you.

**Comment by mf-code** [comment]
[5939] - asJ4MMS3L9A2

**Comment by mf-code** [comment]
[5939] - zkkw8hjFC9ES

**Comment by mf-code** [comment]
[5939] - vOODfr5IJOk2

**Comment by mf-code** [comment]
[5939] - uuArfWsoMcfA

**Comment by mf-code** [comment]
[5939] - openjdk8 - Allure report on Netlify: https://5f47f139ac361391832960e1--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5939] - openjdk10 - Allure report on Netlify: https://5f47f23eaa6f5e68c4302d9d--musing-swartz-6fc060.netlify.app

**Pull request merged**